packages/editor/src/extensions/task-list/task-list.ts

```diff
@@ -49,7 +53,15 @@ export const TaskListNode = TaskList.extend({
     return {
       stats: {
         default: { checked: 0, total: 0 },
-        rendered: false
+        rendered: false,
+        parseHTML: (element) => {
+          if (!element.dataset.title) return { checked: 0, total: 0 };
+          const items = element.querySelectorAll("li.checklist--item");
+          const checked = Array.from(items).filter((item) =>
+            item.classList.contains("checked")
+          ).length;
```

Comment:
This is...unnecessary. Why not just do `querySelectorAll("li.checklist--item.checked").length`?